ci/cron/daily-compat.yml

```
@@ -263,6 +263,18 @@ jobs:
               --query-store-index=oracle
           done
 
+          READ_PERF_KEYS="\
+           fetchByKey \
+           fetchByQuery\
+          "
+          RES=""
+          for KEY in $READ_PERF_KEYS; do
+            perf=$(cat ${OUT}/${KEY}/*/summary.csv | cut -d, -f8 | tail -n 1)
```

I would recommend using `awk` to get the field without relying on a specific length of the line using `cut`. E.g.
```suggestion
            perf=$(cat ${OUT}/${KEY}/*/summary.csv | awk -F, '{print $3}' | tail -n 1)
```
This is just an example and you don't have to merge it as is, but this would mean use `,` as a separator (`-F,`) and print the third column (one-based indexing).

---

Ohhh, yes, you're right sorry, I misread the original command.